player/tests/test.rs

```diff
@@ -186,8 +186,7 @@ impl Test<'_> {
 
             if &expected_data[..] != contents {
                 panic!(
-                    "Test expectation is not met!\nBuffer content was:\n{:?}\nbut expected:\n{:?}",
-                    contents, expected_data
+                    "Test expectation is not met!\nBuffer content was:\n{contents:?}\nbut expected:\n{expected_data:?}"
```

Comment:
context: `rustfmt` refuses to format runs of expressions with lines that exceed the default 100-column limit which it can't break up. We often run into this when we're writing shaders in string literals, or have a long diagnostic message we want to present.

issue(non-blocking): By lengthening this formatting spec. string literal to go past that limit here, we're making `rustfmt` run into this case. 

Since this is already merged, I'm gonna open a new PR and fix this and other cases up: <https://github.com/gfx-rs/wgpu/pull/6349>

I wish this weren't such a hard-to-detect problem; the only reason I recognize it is because I've done this dozens of times to myself.